cmd/poller/poller.go

```diff
@@ -645,20 +654,16 @@ func (me *Poller) load_exporter(name string) exporter.Exporter {
 		return nil
 	}
 
-	binpath = path.Join(me.options.HomePath, "bin", "exporters")
-
-	if sym, err = dload.LoadFuncFromModule(binpath, strings.ToLower(class), "New"); err != nil {
-		logger.Error().Msgf("dload: %v", err.Error())
+	absExp := exporter.New(class, name, me.options, params)
```

Comment:
where did we initialized class var in this method?

Reply:
line 652
if class = params.GetChildContentS("exporter"); class == "" {